chef/automate
# Dan/ingest healthcheck output

### :nut_and_bolt: Description: What code changed, and why?

For the past few versions, habitat has been sending the output of health check commands/scripts with its events, but automate has not ingested/stored this content. This patch allows automate to store the content and updates the data generator to support sending these fields in generated data.

### :chains: Related Resources

### :+1: Definition of Done

### :athletic_shoe: How to Build and Test the Change

* Build applications service, start all services
* insert generated data with `applications_populate_database`
* For now the new fields are not exposed in the API, but you can see the data is ingested if you `chef-automate dev psql chef_applications_service` and then `select * from service_full;`
